server/src/routes/auth.ts

```diff
@@ -427,7 +437,7 @@ router.delete('/delete', authenticate, async (req: Request, res: Response) => {
  * This route will return all expenses for a user, including individual expenses and group expenses where the user has a split.
  * The user must be authenticated to make this request.
  *
- * @route GET /users/:userId/expenses
+ * @route GET /users/expenses
```

Comment:
Because I was talking to victoria and she was telling me that the call is just users/expenses and it just checks with the token, it was late and im sick i can undo it but im just not sure

Reply:
sorry I shouldve been more clearer. to fetch all user's expenses is `user/id/expenses` but to create a new expense is just `/expenses` which is where the token parsing comes in play.